### PR TITLE
Upgrade tesseract to 4.1.1

### PR DESCRIPTION
**Pull request by dimon222**
**Don't merge this until its tested and ready for immediate release** (on various configs including legacy AVX unsupported CPUs)
Side effect of this upgrade - all users will be required to use latest version of WFInfo (9.3.4?), as old dlls will be deleted from `master` branch upon merge. Ideally we will want to consider moving away these dlls to another location outside of github to avoid such breaking changes.

* Upgrade upstream tesseract to 4.1.1 (refer to [this](https://github.com/tesseract-ocr/tesseract/blob/4.1/ChangeLog) for changelog)
* Upgrade dotnet tesseract wrapper to 4.1.1 (refer to [this](https://github.com/charlesw/tesseract/blob/master/ChangeLog.md) for changelog)
* Update leptonica to 1.80 (required transitive dependency)
* Add cleanup script for old dlls to avoid conflicts

How to test while its in PR stage:
1. Checkout to this branch
2. "Clean WFInfo", "Rebuild WFInfo", launch WFInfo, see if it loads fully without crashes, try to process some valid warframe image (perhaps via debug Shift + Activation key)

PS: The rollback to older version of WFInfo will require manual deletion of tesseract4 folder (or dlls inside of it)

### CODE REVIEW

**Comment by dimon222** [comment]
Changed the location of libs, no longer requiring any manual changes to test it.

File: WFInfo/CustomEntrypoint.cs [hunk 1]
```diff
@@ -240,7 +271,7 @@ private static void DownloadProgressCallback(object sender, DownloadProgressChan
             dialogue.Dispatcher.Invoke(() => { dialogue.UpdatePercentage(e.ProgressPercentage); });
         }
 
-        private static async void RefreshTesseractDlls(CancellationToken token)
+        private static async void RefreshTesseractDlls(CancellationToken token, bool AvxSupport)
```
Comment (TobiTenno) [inline WFInfo/CustomEntrypoint.cs:274]: outside perspective, but `AvxSupport` doesn't seem to be used or passed down in here

Comment (dimon222) [reply WFInfo/CustomEntrypoint.cs:274]: It wasn't used in very straightforward way (felt like global scope), so I decided to make it more clear - its local variable at least.
It shouldn't functionally affect anything.

Comment (dimon222) [reply WFInfo/CustomEntrypoint.cs:274]: Its used [here](https://github.com/WFCD/WFinfo/blob/c3441f114748e4bc112e16940cae1cffc812936e/WFInfo/CustomEntrypoint.cs#L303)

**Comment by dapal-003** [comment]
Build without errors and confirmed by running some reward screen and inventories at it. No issues with the code, waiting for korean translation to finalize to merge.

**Comment by dapal-003** [comment]
Finally got around to applying the KO branch changes. It compiled, and ran from scratch perfect here. I'm green lighting it.

**Review by dapal-003 — APPROVED** [review]

**Comment by dimon222** [comment]
Pls don't merge it, I want it to use our libs branch for future upgrades/more langs.

**Comment by dapal-003** [comment]
Ah, I knew I was missing something thanks for reminding me

**Pull request closed without merging**